cmd/hasher.go

```diff
@@ -0,0 +1,18 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"golang.org/x/crypto/bcrypt"
+)
+
+func main() {
+	password := os.Args[1]
+
+	hash, err := bcrypt.GenerateFromPassword([]byte(password), 12)
```

Comment:
The hardcoded cost of 12 is higher than the default but I think in general we should stick to bcrypt.DefaultCost

Reply:
Nice. Didn't know of that.

Reply:
@nwillems: Can you please update your patch to use bcrypt.DefaultCost instead of 12?

Reply:
Should be done by now. Sorry for the delay.